src/components/Sykmeldingview/Layout/JaEntry/JaEntry.tsx

```diff
@@ -0,0 +1,21 @@
+import { Element } from 'nav-frontend-typografi';
+import './JaEntry.less';
+
+interface JaEntryProps {
+    show: boolean;
+    title: string;
+}
+
+const JaEntry: React.FC<JaEntryProps> = ({ show, title }) => {
+    if (show) {
```

Comment:
Ser du bare har fulgt patternet som var brukt i den forrige komponenten, men synes dette patternet er unødvendig.

Fra et lesbarhets-perspektiv:

Med `<JaEntry show={someVal} title={title} />` så kan det jo bety så mangt, returnerer det null, animeres den inn og ut, skjer det noen side effects (hooks etc) selv om show er false?

Mens med `{someVal && <JaEntry title={title} />}` så vet man at der kuttes eksekveringen helt.

Så stemmer for at vi rydder opp det og fjerner show-proppen, flytter den kondisjonelle rendringen opp et nivå. :smile:

Reply:
Ja eg var innom tanken, skal endre det!